src/app/(events)/map/components/GoogleMaps/index.tsx

```diff
@@ -203,6 +212,21 @@ export const GoogleMaps = ({ setMapLoaded }: GoogleMapsProps) => {
       <div className={classes.container}>
         <Calendar date={date} setDate={setDate} />
       </div>
+      {eventAdded && (
+        <div className={classes.overlay}>
+          <Notification
+            style={{ padding: "3rem" }}
+            className={`${classes.notification}`}
+            onClick={handleIconClick}
+            icon={checkIcon}
```

Comment:
@SachiGoto This way `onClick={handleIconClick}` is absolutely ok, but we can also do like this to reduce the code lines

```typescript
onClick={() => setEventAdded(false)}
```

Reply:
Ah that's a really good point! I tend to make things unnecessary complicated haha 
Thank you so much for the suggestion!

Reply:
just updated my code!